chainercv/links/model/pickable_sequential_chain.py

```diff
@@ -0,0 +1,151 @@
+import chainer
+
+
+class PickableSequentialChain(chainer.Chain):
+
+    """A sequential chain that can pick intermediate layers.
+
+    Callable objects, such as :class:`chainer.Link` and
+    :class:`chainer.Function`, can be registered to this chain with
+    :meth:`init_scope`.
+    This chain keeps the order of registrations and :meth:`__call__`
+    executes callables in that order.
+    A :class:`chainer.Link` object in the sequence will be added as
+    a child link of this link.
+
+    :meth:`__call__` returns single or multiple layers that are picked up
+    through a stream of computation.
+    These layers can be specified by :obj:`pick`, which contains
+    the names of the layers that are collected.
+    When :obj:`pick` is a string, single layer is returned.
+    When :obj:`pick` is an iterable of strings, a tuple of layers
+    is returned. The order of the layers is the same as the order of
+    the strings in :obj:`pick`.
+    When :obj:`pick` is :obj:`None`, the last layer is returned.
+
+    Examples:
+
+        >>> import chainer.functions as F
+        >>> import chainer.links as L
+        >>> model = PickableSequentialChain()
```

Comment:
`PickableSequentialChain` is not imported. (Is it OK?)

Reply:
It is a gray zone.
For instance, I left the documentation for VGG16 without import.

Do you think we should always import the class that is documented?
Even if we add the import line, the snippet still does not work because `x` is undefined.

Reply:
> Even if we add the import line, the snippet still does not work because x is undefined.

I see. So it's OK.